fvm/bootstrap.go

```diff
@@ -36,6 +36,9 @@ type BootstrapProcedure struct {
 	storagePerFlow                   cadence.UFix64
 	restrictedAccountCreationEnabled cadence.Bool
 
+	// TODO: restrictedContractDeployment should be a bool after RestrictedDeploymentEnabled is removed from the context
+	restrictedContractDeployment *bool
```

Comment:
Why is this a pointer to a bool, and not just a bool? Maybe add a comment explaining the three different states of this field

Reply:
added in [5221e00](https://github.com/onflow/flow-go/pull/2376/commits/5221e00a23f4b4cf5be62059433e7f38246e8546)